src/aiida/cmdline/commands/cmd_plugin.py

```diff
@@ -48,7 +48,9 @@ def plugin_list(entry_point_group, entry_point):
             echo.echo_critical(str(exception))
         else:
             try:
-                if (inspect.isclass(plugin) and issubclass(plugin, Process)) or plugin.is_process_function:
+                if (inspect.isclass(plugin) and issubclass(plugin, Process)) or (
+                    False if not hasattr(plugin, 'is_process_function') else plugin.is_process_function
+                ):
```

Comment:
This phrasing with a nested `if` feels a bit awkward. I think a more typical formulation would be
```suggestion
                if (inspect.isclass(plugin) and issubclass(plugin, Process)) or (
                    hasattr(plugin, 'is_process_function') and plugin.is_process_function
                ):
```